### ISSUE TITLE
about Error Handling

### ISSUE THREAD

**Issue by jinmarcus** [bcit-ci/CodeIgniter4#778]
`throw new \CodeIgniter\PageNotFoundException('Controller is empty.');

`

抛出的错误始终是：pageNotFound

CodeIgniter.php  lines：
`throw new PageNotFoundException(lang('HTTP.pageNotFound'));`

**lang('HTTP.pageNotFound')**   

Shouldn't it be like this?

**Comment by jinmarcus** [comment]
Can't the problem be solved?

**Comment by lonnieezell** [comment]
Yes, that looks like it would fix the issue. Care to submit a PR?

**Comment by jinmarcus** [comment]
I've submitted a PR.

**Comment by lonnieezell** [comment]
@jinmarcus I don't see any [pull requests](https://github.com/bcit-ci/CodeIgniter4/pulls) for this listed.

**Comment by jinmarcus** [comment]
https://github.com/bcit-ci/CodeIgniter4/pull/792

**Comment by jinmarcus** [comment]
Does this PR have no fixed?

**Comment by jinmarcus** [comment]
@lonnieezell Does this PR have no fixed?#778 & #792

**Comment by lonnieezell** [comment]
Changes were requested in that pull request that were never implemented. I can’t accept the PR until it’s fixed.
